e2e-tests/playwright/e2e/github-happy-path.spec.ts

```diff
@@ -16,18 +16,30 @@ test.describe.skip('GitHub Happy path', () => {
   const component =
     'https://github.com/janus-idp/backstage-showcase/blob/main/catalog-entities/all.yaml';
 
-  test.beforeAll(async ({ browser }) => {
+  const resources = [
+    'Janus-IDP Authors',
```

Comment:
Can we use it externally from the test? Like on Page object page, so we can reuse it in another tests if needed